qa/os/src/test/java/org/elasticsearch/packaging/test/DockerTests.java

```diff
@@ -563,7 +563,7 @@ public void test120DockerLogsIncludeElasticsearchLogs() throws Exception {
         waitForElasticsearch(installation);
         final Result containerLogs = getContainerLogs();
 
-        assertThat("Container logs don't contain abbreviated class names", containerLogs.stdout, containsString("o.e.n.Node"));
+        assertThat("Container logs contain abbreviated class names", containerLogs.stdout, containsString("org.elasticsearch.node.Node"));
```

Comment:
Since the message is printed for faield assertions, maybe we want this?

```suggestion
        assertThat("Container logs should contain full class names", containerLogs.stdout, containsString("org.elasticsearch.node.Node"));
```